### PR TITLE
Enlarge symfony deps version bound

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,11 +15,10 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=7.0",
-        "symfony/console": "^3.1",
-        "symfony/filesystem": "^3.1",
-        "symfony/process": "^3.1",
-        "symfony/stopwatch": "^3.1",
```
Comment (chalasr) [inline composer.json:21]: removed stopwatch btw (see https://github.com/manala/manalize/pull/87#discussion_r89519358)

**Comment by ogizanagi** [comment]
I've almost thought of a spam :trollface: (enlarge your p...rojects)

**Comment by ogizanagi** [comment]
Did you tested it with lowest dependencies? We should update travis too.

**Comment by chalasr** [comment]
> Did you tested it with lowest dependencies?

Yes it works :)

> We should update travis too

Indeed!

**Comment by ogizanagi** [comment]
Also, why do we still require `"symfony/console": "~3.2@dev"` in dev?

**Comment by chalasr** [comment]
For the awesome `CommandTester#setInputs()` :trollface: :)

**Comment by ogizanagi** [comment]
TOTALLY WORTH IT 👍 . Sorry. 😄

**Comment by ogizanagi** [comment]
(I've added the `In Progress` label for keeping in mind we need to update the `.travis.yml` file to test on lowest dependencies too in the matrix.)

File: .travis.yml [hunk 2]
```diff
@@ -4,13 +4,18 @@ language: php
...
 matrix:
     fast_finish: true
     include:
         - php: 7.0
         - php: 7.1
           env: CHECK_PHP_SYNTAX="yes"
+        - php: 7.0
+          env: DEPS="low"
+        - php: 7.1
+          env: DEPS="low"
```
Comment (ogizanagi) [inline .travis.yml]: I think only one build with `env: DEPS="low"` is enough actually.

**Pull request merged**